src/components/NavBar.jsx

```diff
@@ -1,27 +1,45 @@
 import { NavLink, Link } from 'react-router-dom';
+import { useState } from 'react';
 import logo from '../images/planet_1.png';
 
-const NavBar = () => (
-  <header>
-    <nav>
-      <Link to="/">
-        <img className="logo" src={logo} alt="" />
-        <h1>Space traveler&apos; Hub</h1>
-      </Link>
-      <ul>
-        <NavLink className="nav-items" to="/">
-          Rockets
-        </NavLink>
-        <NavLink className="nav-items" to="/missions">
-          Missions
-        </NavLink>
-        <span>|</span>
-        <NavLink className="nav-items" to="/profile">
-          My Profile
-        </NavLink>
-      </ul>
-    </nav>
-  </header>
-);
+const NavBar = () => {
+  const [active, setActive] = useState('Rockets');
+
+  return (
+    <header>
+      <nav>
+        <Link to="/">
+          <img className="logo" src={logo} alt="" />
+          <h1>Space traveler&apos; Hub</h1>
+        </Link>
+        <ul>
+          <NavLink
+            className={active === 'Rockets' ? 'active' : ''}
+            to="/"
+            onClick={() => setActive('Rockets')}
+          >
+            Rockets
+          </NavLink>
+          <span>|</span>
+          <NavLink
+            className={active === 'Missions' ? 'active' : ''}
+            to="/missions"
+            onClick={() => setActive('Missions')}
+          >
+            Missions
+          </NavLink>
+          <span>|</span>
```

Comment:
[Optional] Good to use a vertical separator. But I suggest you to manage this with CSS border style to avoid adding a span element and minimize the hierarchy of the DOM structure.